### ISSUE TITLE
TW stats - incorrect number of citations from the last week

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#3020]
The stats show 77 citations, I know that around one thousands new citations were created during last week.
![image](https://user-images.githubusercontent.com/5597748/173202146-5d40b09e-9c68-4cb7-9aae-1cdcd1c710a6.png)

**Comment by mjy** [comment]
The week is rolling (i.e. the exact last 7 days), not calendar.  Knowning this is it still a bug?  Remember numbers will scroll down as the window rolls on.

**Comment by proceps** [comment]
Yes, I have a student working on Asserted distribution, she created a lot of citation, I can see those if I use project activity (citations)
